tmt/steps/prepare/install.py

```diff
@@ -238,3 +246,21 @@ def go(self, guest):
             guest.execute(
                 f"{check} || {command} install {options} "
                 f"{packages}{yum_check}")
+
+        # Install debug{info,source} from repos
+        if repo_debuginfos:
+            # Show a brief summary by default
+            if not self.opt('verbose'):
+                summary = fmf.utils.listed(repo_debuginfos, max=3)
+                self.info('package', summary, 'green')
+            # Provide a full list of packages in verbose mode
+            else:
+                summary = fmf.utils.listed(repo_debuginfos, 'package')
+                self.info('package', summary + ' requested', 'green')
+                for package in sorted(repo_debuginfos):
+                    self.verbose(package, shift=1)
```

Comment:
This summary part code is almost completely duplicated from above. Let's create a new private method `_package_summary` and move the shared code for both `repo_packages` and `debuginfo_packages` there.

Reply:
Agreed, will do.

Reply:
Done, thanks